### PR TITLE
Hot fix not exporting first row of data

### PR DESCRIPTION
**Pull request by edmofro**
### Issue #:
Broke this during my hot fix earlier in the week https://github.com/beyondessential/tupaia/pull/3124
🤦 🤦 🤦

### CODE REVIEW

File: packages/meditrak-server/src/apiV2/export/exportSurveyResponses/exportResponsesToFile.js [hunk 1]
```diff
@@ -241,7 +241,7 @@ export async function exportResponsesToFile(
       surveyResponses.map((response, index) => [response.id, index]),
     );
     answers
-      .filter(answer => !!questionIdToIndex[answer['question.id']]) // filter out answers for a non-exported question, e.g. DateOfData
+      .filter(answer => questionIdToIndex[answer['question.id']] !== undefined) // filter out answers for a non-exported question, e.g. DateOfData
```
Comment (rohan-bes) [inline packages/meditrak-server/src/apiV2/export/exportSurveyResponses/exportResponsesToFile.js:244]: `Falsey` logic is the worst design decision in the javascript language, change my mind 😝

**Review by rohan-bes — APPROVED** [review]
Nice! Let's all chat to work out the hotfix merging strategy 👍

File: packages/meditrak-server/src/apiV2/export/exportSurveyResponses/exportResponsesToFile.js (hunk 1 above)
Comment (edmofro) [reply packages/meditrak-server/src/apiV2/export/exportSurveyResponses/exportResponsesToFile.js:244]: Ha I'm totally with you!

Comment (nick-beyondessential) [reply packages/meditrak-server/src/apiV2/export/exportSurveyResponses/exportResponsesToFile.js:244]: !wrong

**Pull request merged**